tests/appsec/test_automated_login_events.py

```diff
@@ -47,10 +49,14 @@ def test_login_pii_success(self):
             assert r.status_code == 200
             for _, _, span in interfaces.library.get_spans(request=r):
                 meta = span.get("meta", {})
+                if context.library == "nodejs":
+                    assert meta["usr.id"] == " "
```

Comment:
Node sets it to a space?
Sounds like a bug rather than something we should hardcode here?
cc @uurien

Reply:
it's related to[ this discussion](https://dd.slack.com/archives/C054NVD6URE/p1688468402933179?thread_ts=1688053198.321579&cid=C054NVD6URE)

Reply:
Could you also write a small comment explaning why nodejs behave differently ?